test/message_test.rb

```diff
@@ -0,0 +1,30 @@
+require "minitest/autorun"
+require "minitest/pride"
+require "./lib/morse_translator"
+require "./lib/message"
+
+class MessageTest < Minitest::Test
+  def test_message_exists
+    morse_translator = MorseTranslator.new
+    message = Message.new("There are 3 ships", morse_translator)
+    assert_instance_of Message, message
+  end
+
+  def test_message_has_translator
+    morse_translator = MorseTranslator.new
+    message = Message.new("There are 3 ships", morse_translator)
+    assert_instance_of MorseTranslator, message.translator
```

Comment:
Instead of asserting it's a MorseTranslator object, you should assert exactly what object it is i.e.

```ruby
assert_equal morse_translator, message.translator
```